lib/libzutil/zutil_import.c

```diff
@@ -1005,27 +1005,32 @@ zpool_read_label(int fd, nvlist_t **config, int *num_labels)
 	}
 
 	if (lio_listio(LIO_WAIT, aiocbps, VDEV_LABELS, NULL) != 0) {
+		/*
+		 * A portion of the requests may have
+		 * been submitted. Clean them up.
+		 */
 		int saved_errno = errno;
-		errno = 0;
-		int do_slow = 0;
+		boolean_t do_slow = B_FALSE;
 		error = -1;
 
-		for (l = 0; l < VDEV_LABELS; l++) {
-			switch (aio_error(&aiocbs[l])) {
-				case 0:
-					break;
-				case -1:
-					break;
-				case EOPNOTSUPP:
-					do_slow = 1;
-				case EIO:
-				case EINTR:
-				case EAGAIN:
-					/*
-					 * A portion of the requests may have
-					 * been submitted. Clean them up.
-					 */
-					(void) aio_return(&aiocbs[l]);
+		if (saved_errno == EAGAIN || saved_errno == EINTR ||
```

Comment:
At this point saved_errno and errno should be identical.  So you can eliminate the line wrap by using plain `errno`.

Reply:
Good point. Corrected.